.github/workflows/windows-package.yml

```diff
@@ -52,10 +52,8 @@ jobs:
     - run:
           python setup.py bdist_wheel
       if: ${{ github.event_name == 'release' }}
-    - name: Install twine
-      run: python -m pip install twine
-    - name: upload wheels
-      env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
-      run: twine upload  dist/* 
+           python -m pip install twine
+      env:      
+            TWINE_USERNAME: __token__
+            TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+    -  run:  twine upload  dist/* 
```

Comment:
even this run should be combined. The error happened in the build (you can check) because this command ran whereas the previous command `python setup.py bdist_wheel` did not run. Therefore, there was nothing to upload. Therefore, the run command must first run `bdist_wheel` and then install twine and then run `twine upload`